Project.toml

```diff
@@ -1,23 +1,24 @@
 name = "FiniteDifferences"
 uuid = "26cc04aa-876d-5657-8c51-4c34ba976000"
-version = "0.11.7"
+version = "0.12.0"
```

Comment:
Why is this breaking?

Reply:
I figured that a minor version increase was warranted because the internals now work very differently, whic affects the behaviour of the estimates (hopefully in a good way). One breaking change is that the keyword argument `max_step` is removed.